contracts/modules/cover/Cover.sol

```diff
@@ -182,7 +182,6 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     require(params.period >= MIN_COVER_PERIOD, "Cover: Cover period is too short");
     require(params.period <= MAX_COVER_PERIOD, "Cover: Cover period is too long");
     require(params.commissionRatio <= MAX_COMMISSION_RATIO, "Cover: Commission rate is too high");
-    require(params.amount > 0, "Cover: amount = 0");
```

Comment:
We should keep this require statement because the allocations array could be empty and the final cover amount will end up being zero.